packages/stacked_tools/lib/src/services/file_service.dart

```diff
@@ -94,7 +94,7 @@ class FileService {
     await writeFile(
       file: File(filePath),
       fileContent: fileLines.join('\n'),
-      type: FileModificationType.Modify,
+      type: FileModificationType.ApplyModification,
```

Comment:
Nice naming

Reply:
Thanks! I did that cause it wasn't easy to determine when the logs should say reverted and when they should say applied modification with that old enum